final_task/rss_reader/string_operations.py

```diff
@@ -0,0 +1,64 @@
+
+def cut_string_to_length_with_space(basic_str, length):
+    """function that cut given string to the list of strings with a given max length"""
+    my_strings = []
+    my_str = ''
+    for i, word in enumerate(basic_str.split()):
+        my_str += word + ' '
+        if i + 1 < len(basic_str.split()):
+            next_word = basic_str.split()[i+1]
+        else:
+            next_word = ''
+            my_strings.append(my_str)
+        if len(my_str + next_word) >= length:
+            my_strings.append(my_str)
+            my_str = ''
+    return my_strings
+
+
+def make_string_readable(basic_str):
+    """return copy of string, replace &#39; to ' and &quot; to " """
+    my_str = basic_str[:]
+    my_str = my_str.replace('&quot;', '\"')
+    my_str = my_str.replace('&#39;', "\'")
+    return my_str
```

Comment:
It is better to look for more universal solution in some libraries. There may be other symbols that break your encoding,

Reply:
Fixed